Resources/doc/suggesters/usage.rst

```diff
@@ -0,0 +1,87 @@
+Suggesters Usage
+================
+
+`Suggesters`_ are used to find a similar looking term using the provided text.
+
+You can use suggesters in any query as you would normally, or `suggester api`_.
```

Comment:
You can use suggesters (in any query, as you would normally) or `suggester api`_.